package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "find-root": "^1.0.0",
     "loophole": "^1.1.0",
+    "minimatch": "^3.0.3",
```

Comment:
FYI new dependency: I figured it was better to add the dependency then reinvent the wheel and try to make my own glob regexes or something.